embedded-workshop-book/src/generating-pac.md

```diff
@@ -0,0 +1,67 @@
+# Generating the PAC
+
+Generate your own PAC from an SVD file. 
+
+## In this exercise you will learn how to
+* generate a PAC from an SVD file.
+* format the generated code. 
+* split the single PAC file into it's modules.
+
+## Prerequisites
+* usage of cargo install
+* generating docs
+
+## Tasks
+* Install `svd2rust` and `form` via cargo. 
+* Download the [nrf-svd][svd] file and place it into `down-the-stack/dk-pac`.
+* Run `svd2rust` on the file using the `cortex-m` flag. 
+* Format the generated file to make it readable. 
+* Split the file into its modules using `form`.
```

Comment:
Should these be the other way around? Or should you format, run form, and then format again? Not sure, haven't tried in a long while.

Reply:
they should